HelloWorld.scala

```diff
@@ -1,3 +1,4 @@
+import org.apache.spark.sql.DataFrame
```

Comment:
please utilize big data to println Helloworld, look at https://www.programcreek.com/scala/org.apache.spark.SparkConf

try to add

```
val conf:SparkConf = new SparkConf().setAppName("Histogram").setMaster("local")
val sc: SparkContext = new SparkContext(conf)
```

then add http://www.scalatest.org/getting_started_with_fun_spec that adds the spark dataframe

Reply:
Are you sure an RDD isn't better for this?

Reply:
it'll be a learning experience for Bishoy